UltiSnips/snippets.snippets

```diff
@@ -2,8 +2,8 @@ priority -50
 
 # We use a little hack so that the snippet is expanded
 # and parsed correctly
-snippet snip "Snippet definition" b
-`!p snip.rv = "snippet"` ${1:Tab_trigger} "${2:Description}" ${3:b}
+snippet snip "Snippet definition" bs
```

Comment:
Ah. Good catch. I don't think this is actually required, since the second tabstop encompasses the space between 3 and 4. Should I just remove the `s`?